JSONWebToken.podspec

```diff
@@ -5,12 +5,12 @@ Pod::Spec.new do |spec|
   spec.homepage = 'https://github.com/kylef/JSONWebToken.swift'
   spec.license = { :type => 'BSD', :file => 'LICENSE' }
   spec.author = { 'Kyle Fuller' => 'kyle@fuller.li' }
-  spec.source = { :git => 'https://github.com/kylef/JSONWebToken.swift.git', :tag => "#{spec.version}" }
+  spec.source = { :git => 'https://github.com/kevin-chen-bcgdv/JSONWebToken.swift.git', :branch => "swift-3" }
   spec.source_files = 'Sources/*.swift'
   spec.ios.deployment_target = '8.0'
   spec.osx.deployment_target = '10.9'
   spec.tvos.deployment_target = '9.0'
   spec.requires_arc = true
-  spec.dependency 'CryptoSwift', '~> 0.4.0'
+  spec.dependency 'CryptoSwift'
```

Comment:
This basically means that people can use unsupported versions of CryptoSwift with JSONWebToken. Along with if there are future incompatible releases of CryptoSwift break JSONWebToken for users in the future.

We should always pin to the API compatible version.